apps/auth/main.py

```
@@ -1,16 +1,18 @@
-from fastapi import FastAPI, Form
+from fastapi import FastAPI, HTTPException, Response
 import uvicorn
 import models
-from config import PASSWORD_MAX_LENGTH, PASSWORD_MIN_LENGTH
+from config import PASSWORD_MAX_LENGTH, PASSWORD_MIN_LENGTH, AUTH_PORT, ISSECURE
 import database as db
```

Ruff (0.8.2)

`database` imported but unused

Remove unused import: `database`

(F401)